x-pack/legacy/plugins/uptime/public/components/functional/charts/duration_chart.tsx

```diff
@@ -122,10 +87,13 @@ export const DurationChart = ({
                 defaultMessage: 'Duration ms',
               })}
             />
-            {lineSeries}
+            {hasLines && (
+              <DurationLineSeriesList lines={locationDurationLines} meanColor={meanColor} />
+            )}
+            {!hasLines && <DurationChartEmptyState />}
```

Comment:
i think using ternary operator will be more readable , though it depends on developer preference, i personally find ternary operator more readable :)

Reply:
Touché, I agree! Added in https://github.com/elastic/kibana/pull/45734/commits/2a2ce0c73277587ea05c8eef018ad2fa05aa5d69.